README.md

```diff
@@ -1,21 +1,26 @@
-# gateway-certs-generator
+# Certificate Generator for Slice Gateway
 
-This is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to easy-rsa (typically bundled with OpenVPN).
+The `gateway-certs-generator` is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to `easy-rsa` (typically bundled with OpenVPN).
```

Comment:
change it to It as it's the first word of a new sentence.

Reply:
updated